src/Robo/Plugin/Commands/DevelopmentModeCommands.php

```diff
@@ -24,15 +24,48 @@ class DevelopmentModeCommands extends DevelopmentModeBaseCommands
      *
      * @param string $siteName
      *   The Drupal site name.
+     * @option skip-lando-start
+     *   Skip starting Lando.
      *
      * @aliases magic
      *
      * @return \Robo\Result
      *   The result of the set of tasks.
      */
-    public function devRefresh(string $siteName = 'default'): Result
+    public function devRefresh(string $siteName = 'default', array $options = ['skip-lando-start' => false]): Result
     {
-        return $this->devRefreshDrupal($siteName);
+        return $this->devRefreshDrupal($siteName, $options['skip-lando-start']);
+    }
+
+    /**
+     * Refreshes development environments for *all* sites.
+     *
+     * Completely refreshes a development environment including running 'composer install', starting Lando, downloading
+     * a database dump, importing it, running 'drush deploy', disabling front-end caches, and providing a login link.
+     *
+     * Examples:
+     *   dev:refresh-all --skip-sites=common,example --skip-lando-start
+     *
+     * @option skip-sites
+     *   A comma separated list of sites to skip.
+     * @option skip-lando-start
+     *   Skip starting Lando.
+     *
+     * @return \Robo\Result
+     *   The result of the set of tasks.
+     */
+    public function devRefreshAll(array $options = ['skip-sites' => '', 'skip-lando-start' => false]): Result
+    {
+        $skipLandoStart = $options['skip-lando-start'];
+        $siteNames = $this->getAllSiteNames();
+        $result = null;
+        foreach ($siteNames as $siteName) {
+            if (in_array($siteName, explode(',', $options['skip-sites']), true)) {
```

Comment:
> (I didn't even know that was an option and had to google it to understand what it even did.)

@adamzimmermann Did you find something because this was just my creation to be able to [skip a site on Benz](https://github.com/ChromaticHQ/benz-platform/blob/e169c6736974f33b2ea3fcece7192de34051a718/.sites.config.yml#L61-L66). 😆

Reply:
I was referring to the strict checking option that the third argument provides.

Reply:
Ohhhh. Hah! I learned about it myself doing this. New warnings teaching us things!